bundles/org.openhab.binding.tplinksmarthome/src/main/java/org/openhab/binding/tplinksmarthome/internal/device/BulbDevice.java

```diff
@@ -103,7 +103,7 @@ private HasErrorResponse handleDecimalType(String channelID, Connection connecti
         return null;
     }
 
-    private TransitionLightStateResponse handleColorTemperature(Connection connection, int colorTemperature,
+    private @Nullable TransitionLightStateResponse handleColorTemperature(Connection connection, int colorTemperature,
```

Comment:
As the `connection` property has been moved to the super class `SmartHomeDevice` can it be removed here - and elsewhere?

Reply:
Yes makes sense. I've removed the `connection` and `configuration` parameters.

Reply:
ok got merged before I could add :smile: (I've create a new pr: #5766)